### PR TITLE
Update test cases

### PR DESCRIPTION
**Pull request by liuw1**
tdinfo: gpaw value can only be 48 or 52
trustedboot: update log format

### CODE REVIEW

File: tests/test-td-payload/src/testtdinfo.rs [hunk 1]
```diff
@@ -48,11 +48,10 @@ impl TestCase for Tdinfo {
      */
     fn run(&mut self) {
         let mut td_info = tdx::tdcall_get_td_info().expect("Failt to get td info");
-
-        if (self.expected.gpaw != td_info.gpaw) {
```
Comment (jyao1) [inline tests/test-td-payload/src/testtdinfo.rs:52]: please remove expected.gpaw, if it is not needed.

Comment (liuw1) [reply tests/test-td-payload/src/testtdinfo.rs:52]: Resolved

File: tests/test-td-payload/config/test_config_5.json [hunk 2]
```diff
@@ -73,22 +57,22 @@
         "result": "None",
         "run": true 
     },
-    "tcs007": {
+    "tcs006": {
```
Comment (jyao1) [inline tests/test-td-payload/config/test_config_5.json:60]: please dont change index/name.

Comment (liuw1) [reply tests/test-td-payload/config/test_config_5.json]: Resolved

**Review by jyao1 — APPROVED** [review]

**Pull request merged**